x-pack/plugins/ux/public/services/data/getEsFilter.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ESFilter } from '@kbn/core/types/elasticsearch';
+import { ENVIRONMENT_ALL } from '../../../common/environment_filter_values';
+import {
+  uxLocalUIFilterNames,
+  uxLocalUIFilters,
+} from '../../../common/ux_ui_filter';
+import { UxUIFilters } from '../../../typings/ui_filters';
+import { environmentQuery } from '../../components/app/rum_dashboard/local_uifilters/queries';
+
+export function getEsFilter(uiFilters: UxUIFilters, exclude?: boolean) {
```

Comment:
Likewise copied this from the server because the component's query generator depends on it.

Reply:
Copying is probably fine. Fewer dependencies <3